test_psa_target.py

```diff
@@ -132,10 +141,13 @@ def _erase_flash_storage(args, suite):
     Creates a target specific binary which has its ITS erased
     :param args: Command-line arguments
     :param suite: Test suite
+    :return: return binary name generated 
     """
     mbed_os_dir = join(ROOT, "BUILD", args.mcu, TC_DICT.get(args.toolchain))
     cmd = []
 
+    binary_name = "mbed-os-tf-m-regression-tests-reset-flash-{}.hex".format(suite)
```

Comment:
I think we should rename this binary to something simpler but this is not the point of this PR!